charts/flyte-core/values.yaml

```diff
@@ -945,6 +945,9 @@ external_events:
 # Cloud events are used to send events (unprocessed, as Admin see them) in cloud event format to
 # an SNS topic (or gcp equivalent)
 cloud_events:
+  # -- The name of the secret to use to alternatively load in cloud events configuration via a secret. Useful when the
+  # configuration contains secrets.
+  secretName: ""
```

Comment:
It's for the password, right? If so, could we read the password from the secret?
we did the similar thing for db [here](https://github.com/flyteorg/flyte/blob/fa274e1fa59d0500a8d2d5926d170ca7cbef37ac/datacatalog/pkg/runtime/application_config_provider.go#L29-L36)

Reply:
Yeah it is for the password and I was thinking this would work for any future cloud events types that have secrets in the future. I'll look into what you linked